app.js

```diff
@@ -139,6 +176,29 @@ nextMarker = argv.marker || null;
 AWS.config.sslEnabled = argv.sslEnabled;
 var s3 = new AWS.S3();
 
+if (argv.listFile) {
+  listFile = new LineStream();
+  
+  if (argv.listFile === '-') {
+    process.stdin.pipe(listFile);
+  } else {
+    fs.createReadStream(argv.listFile, {flags: 'r'}).pipe(listFile);
+  }
+  
+  // Read list of objects to copy from a file
+  listFile.on('data', function (line) {
+    var newObject = {
+      Key: line.toString(),
+      StorageClass: 'STANDARD',
+      Size: 0
+    };
+    if (checkKeyMatch(newObject)) {
+      objectsToCopy.push(newObject);
+    }
+    startRequests();
```

Comment:
Yes, this actually isn't for start-up purposes, it's for when things are fully running.  It's equivalent to line 77 above where we do the same thing when S3 list requests are returned.  We need to kick the loop when we get fresh data to start normal copy requests.